src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/temporary/TemporaryNode.java

```diff
@@ -0,0 +1,62 @@
+package org.openstreetmap.atlas.geography.atlas.raw.slicing.temporary;
+
+import java.io.Serializable;
+
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.openstreetmap.atlas.geography.Location;
+import org.openstreetmap.atlas.geography.atlas.items.Node;
+
+/**
+ * The {@link TemporaryNode} object, keeps track of the bare minimum information required to create
+ * an Atlas {@link Node}. It is meant to be as light-weight as possible, keeping track of only the
+ * identifier and location.
+ *
+ * @author mgostintsev
+ */
+public class TemporaryNode implements Serializable
```

Comment:
Is there a reason why this needs to be Serializable and the TemporaryEdge not?

Reply:
TemporaryEdge should be as well.

Reply:
It is - TemporaryEdge extends TemporaryEntity, which is Serializable.

Reply:
doh! good point!